packages/rich-text-structure/src/index.js

```diff
@@ -0,0 +1,13 @@
+export { applyFormat } from './apply-format';
+export { concat } from './concat';
+export { create, createWithSelection } from './create';
+export { getActiveFormat } from './get-active-format';
+export { getTextContent } from './get-text-content';
+export { isCollapsed } from './is-collapsed';
+export { isEmpty } from './is-empty';
+export { isSelectionEqual } from './is-selection-equal';
+export { removeFormat } from './remove-format';
+export { splice } from './splice';
+export { split } from './split';
+export { toDOM, apply } from './to-dom';
```

Comment:
It looks like `toDOM` doesn't need to be exposed as it is used internally by `toString`.

Reply:
`apply` seems to be specific to TinyMCE implementation of `RichText, maybe it should be part of the `RichText` component rather than structure related methods?

Ideally this package should contain platform independent helpers only, do you think this is possible?

Reply:
There is no TinyMCE specific implementation. This is all TinyMCE agnostic. `apply` applies an in memory DOM tree to a live one, so I think it does belong here.

Reply:
`toString` now no longer uses the DOM.